app/api/v_sphere/virtual_machine.rb

```diff
@@ -294,7 +294,7 @@ def users
       rescue Git::GitExecuteError => e
         Rails.logger.error(e)
       end
-      users
+      users || []
```

Comment:
Maybe add a comment here explaining why this change is necessary, because users is already set to [].

Reply:
Moved || [] to the line where nil is assigned to users. Think it is clear enough without a comment now